cfp.md

```diff
@@ -67,9 +67,9 @@ presentation in cases of doubt about your content.
 
 ## Important Dates
 
-* End of the Call for Participation: **December 10th, 2015**
-* Announcement of accepted proposals: **December 17th, 2015**
-* FOSDEM 2016: **January 30th 31st, 2016**
+* End of the Call for Participation: **December 8th, 2016**
```

Comment:
it's very short, we need at least one week to get confirmation that ppl will come or not

I'd do the 4th.